src/pages/preferences/PreferencesTabs.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Skeleton } from "antd";
+import { H1 } from "components/Typography/index";
+import { ProfileTab } from "./ProfileTab";
+import { UserSettings } from "gql/generated/types";
+interface PreferenceTabsProps {
+  tabKey: string;
+  userSettings: UserSettings;
+  loading: boolean;
+  error: any;
+}
+export const PreferencesTabs: React.FC<PreferenceTabsProps> = ({
+  tabKey,
+  userSettings,
+  loading,
+  error,
+}) => {
+  return (
+    <Container>
+      <H1>{mapUrlTabToTitle[tabKey]}</H1>
```

Comment:
the header typography components are deprecated ever since we began using leafygreen typography